client/src/components/AboutMe/CreateAboutMe.js

```diff
@@ -17,9 +20,20 @@ export default function CreateAboutMe() {
         .catch((err) => console.log(err));
   };
 
-  const handleEditorChange = (content) => {
-    setAboutMe(content);
-  };
+  /* const handleEditorChange = (newValue) => {
+    setAboutMe(newValue);
+  }; */
+
```

Comment:
I see that it is a new configuration, we can try both to see which one works better and configure the components that use it with the same tool for the whole project.